eotsmanager/cmd/eotsd/daemon/keys.go

```diff
@@ -0,0 +1,172 @@
+package daemon
+
+import (
+	"bufio"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/client/input"
+	"github.com/cosmos/go-bip39"
+	"github.com/jessevdk/go-flags"
+	"github.com/urfave/cli"
+
+	bbntypes "github.com/babylonchain/babylon/types"
+	"github.com/babylonchain/finality-provider/eotsmanager"
+	"github.com/babylonchain/finality-provider/eotsmanager/config"
+	"github.com/babylonchain/finality-provider/log"
+)
+
+type KeyOutput struct {
+	Name      string `json:"name" yaml:"name"`
+	PubKeyHex string `json:"pub_key_hex" yaml:"pub_key_hex"`
+	Mnemonic  string `json:"mnemonic,omitempty" yaml:"mnemonic"`
+}
+
+var KeysCommands = []cli.Command{
+	{
+		Name:     "keys",
+		Usage:    "Command sets of managing keys for interacting with BTC eots keys.",
+		Category: "Key management",
+		Subcommands: []cli.Command{
+			AddKeyCmd,
+		},
+	},
+}
+
+var AddKeyCmd = cli.Command{
+	Name:  "add",
+	Usage: "Add a key to the eots BTC keyring. Note that this will change the config file in place.",
```

Comment:
```suggestion
	Usage: "Add a key to the EOTS manager keyring.",
```
I can see only the `keyring-backend` will be changed in place. Can we add the note there?

Reply:
done at https://github.com/babylonchain/finality-provider/pull/308/commits/a40dac1946850ce44ce6df4c3bef93f1a662ebbd